apps/web/lib/features/team-members-kanban-view.tsx

```diff
@@ -33,14 +96,6 @@ export const KanbanView = ({ kanbanBoardTasks }: { kanbanBoardTasks: IKanban })
 		tasks.splice(endIndex, 0, removedTask);
 		return tasks;
 	};
-
-	const reorderColumn = (list: IKanban, startIndex: number, endIndex: number) => {
```

Comment:
@Anishali2 why you removed this? It's not needed!?

Reply:
Hm, where is code that you did for reorderColumn? Why we have that code and now you not need it? It seems was related to ability to remove some columns!?